src/services/mortys.js

```diff
@@ -0,0 +1,6 @@
+const getCharacter = (id) => {
```

Comment:
This fetch should be in the rickAndMortyApi.js
Also it's fetching by id so maybe rename 'fetchCharacterById' or something to that affect.